### PR TITLE
fix(bazel): Always disable the wrapper script

### LINKED ISSUE #8828: Bazel: disable Bazelisk optional wrapper script when fetching the version of Bazel
Bazelisk (https://github.com/bazelbuild/bazelisk) has a mechanism to call a wrapper script if it exists.
One of our projects has such setup and it fails when fetching the version:

```
❯ bazel --version 
[FATAL 06:57:24.976 src/main/cpp/blaze.cc:1104] Unknown startup option: '--version'.
  For more info, run 'bazel help startup_options'.
```

The solution is to disable the wrapper script:

```
❯ BAZELISK_SKIP_WRAPPER=true bazel --version
bazel 7.1.0
```

IMO when ORT fetches the version, the wrapper should always be disabled.

### PR DESCRIPTION
**Pull request by fviernau**
Fixes #8828.

### CODE REVIEW

**Comment by fviernau** [comment]
@nnobelis mind testing if this works?

**Comment by nnobelis** [comment]
It does not build for me:

```
/src/main/kotlin/Bazel.kt:83:21 Redundant spread (*) operator.
```

**Comment by nnobelis** [comment]
Yes it works, thanks.

File: plugins/package-managers/bazel/src/main/kotlin/Bazel.kt [hunk 1]
```diff
@@ -77,6 +78,16 @@ class Bazel(
...
     override fun command(workingDir: File?) = "bazel"
 
+    override fun run(vararg args: CharSequence, workingDir: File?, environment: Map<String, String>): ProcessCapture =
+        super.run(
+            args = args,
+            workingDir = workingDir,
+            environment = environment.toMutableMap().apply {
+                // Disable the optional wrapper script under `tools/bazel`, to ensure the --version option works.
+                put("BAZELISK_SKIP_WRAPPER", "true")
+            }
```
Comment (sschuberth) [inline plugins/package-managers/bazel/src/main/kotlin/Bazel.kt:88]: I propose to use the shorter `environment + mapOf("BAZELISK_SKIP_WRAPPER" to "true")`.

**Review by sschuberth — APPROVED** [review]

**Comment by sschuberth** [comment]
The `NuGetFunTest` failure is unrelated.

**Pull request merged**